kuma/static/js/payments-manage.js

```diff
@@ -0,0 +1,27 @@
+
+(function(win, doc) {
+    'use strict';
+
+    var path = win.location.pathname;
+    if (!path.includes('/payments/recurring/management')) {
+        return;
+    }
+
+    var toggleConfirmationButtons = Array.from(doc.querySelectorAll('button[toggle-confirmation]'));
+    var confirmationDialog = doc.getElementById('delete-confirmation');
+
+    /**
+     * Toggle the visibility of the confirmation dialog
+     */
+    function toggleDeleteConfirmaiton() {
+        toggleConfirmationButtons[0].classList.toggle('hidden');
+        confirmationDialog.classList.toggle('hidden');
+        // var hidden = confirmationDialog.getAttribute('aria-hidden');
+        confirmationDialog.toggleAttribute('aria-hidden');
+    }
+
+    toggleConfirmationButtons.forEach(function(button) {
+        button.addEventListener('click', toggleDeleteConfirmaiton);
+    });
```

Comment:
Nit: Up to you, but I would prefer if we delegate the `eventListener` to the container instead of attaching `eventListeners` to each button. Not a huge deal here as there are only a few instances, but generally a good idea.

Reply:
I agree, but in this case it's just one or two buttons. But this is definitely something to consider once we improve the manage payments section, as more logic would be needed.